### ISSUE TITLE
use correct git practises 

### ISSUE THREAD

**Issue by moham96** [150balbes/Amlogic_s905-kernel#4]
Hi @150balbes ,
thanks for your good work on Amlogic platform.It's a bit hard to follow your work on git hub since your commits don't have a meaningful names and comments, also you seem to commit your updates from Torvalds tree without using rebase which mixes the commits.
If you could please use correct git practices (add Torvalds remote to your local repo, use git pull to merge upstream changes and also add meaningful and memorable commits comments instead of just the date ) it would make it easy for us to benefit from your work.

Thanks

**Comment by 150balbes** [comment]
Hi
This thread is designed to quickly build the kernel for the project Armbian and for easy Assembly on the devices themselves S9xxx with the addition of its driver or configuration (without having to put in hand a lot of patches). This is a fixed slice of sources with a working version of the kernel. It is not intended for studying and debugging patches. If you need separate patches that were used in the build - you need to refer to the Libreelec git build system. there patches and kernel source code are separated.
